vip-publication/src/main/java/fr/insalyon/creatis/vip/publication/client/view/Data.java

```diff
@@ -29,15 +29,12 @@
  * The fact that you are presently reading this means that you have had
  * knowledge of the CeCILL-B license and that you accept its terms.
  */
-package fr.insalyon.creatis.vip.core.client.view.user.publication;
+package fr.insalyon.creatis.vip.publication.client.view;
```

Comment:
Is it not necessary to add vipApplication in this class ? I'm lost with all those gwt grid data holder classes.

Reply:
Done